test/controllers/EventReportControllerSpec.scala

```diff
@@ -64,5 +69,34 @@ class EventReportControllerSpec extends SpecBase {
   }
 
 
+  "compileEventOneReport" must {
+
```

Comment:
there is a commented test case, are you planning to remove it or keep it?

Reply:
This test is now implemented on 7145: https://github.com/hmrc/pension-scheme-event-reporting-stubs/pull/3/files#diff-6c0ebde16e61017b8413d14ee61ebc6ea30fb1d97c6f7e28536a1760cc569033

The schema validation wasn't part of this ticket, so it was left out at this stage, and the work continued on 7145.